src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -871,12 +877,11 @@ protected void addVoice(String name, String gender, String lang, Object voicePro
       genderIndex.put(gender, group);
     }
   }
-  
+
```

Comment:
are you using the mrl_java_formatter.xml  file in eclipse ?

Reply:
yes..